lxd/devices.go

```diff
@@ -184,16 +184,21 @@ func deviceTaskBalance(d *Daemon) {
 	// Get effective cpus list - those are all guaranteed to be online
 	effectiveCpus, err := cGroupGet("cpuset", "/", "cpuset.effective_cpus")
 	if err != nil {
-		shared.Log.Error("Error reading host's cpuset.effective_cpus")
-		return
+		// Older kernel - use cpuset.cpus
+		shared.Log.Warn("Failed reading host's cpuset.effective_cpus, falling back to cpuset.cpus")
```

Comment:
Can we drop that warning? On a busy system, deviceTaskBalance may hit every couple of seconds, so printing a warning for what's basically normal operation on old kernel seems a bit excessive.